src/Button.vue

```diff
@@ -243,14 +254,21 @@ $margin: 3px;
     .v-switch-button {
       display: block;
       position: absolute;
-      overflow: hidden;
 
       top: 0;
       left: 0;
 
       transform: translate3d($margin, $margin, 0);
       border-radius: 100%;
       background-color: #fff;
+      
+    }
+    .v-switch-button-text{
+      vertical-align: sub;
+      border-radius: 40%;
+      line-height: 2em;
```

Comment:
`2em` stands for 2 x current font size, seems to be wrong for resizable component

also why border-radius 40% ?

Reply:
Border radius 100% will create a eclipce, needed ton reduce to a shape the is straight but curved at the end.

Line height - cannot use percent 100%.
Suggestions are welcomed...